azurerm/resource_arm_kubernetes_cluster.go

```diff
@@ -1261,3 +1271,20 @@ func flattenKubernetesClusterKubeConfigAAD(config kubernetes.KubeConfigAAD) []in
 
 	return []interface{}{values}
 }
+
+func expandApiServerAuthorizedIPRangeInterfaces(d *schema.ResourceData) *[]string {
```

Comment:
we should do `d.GetOk` and pass the value into this function instead of doing it here.